### PR TITLE
S610-004: Implement is_called_by

### PR DESCRIPTION
**Pull request by Roldak**

### CODE REVIEW

File: ada/language/ast.py [hunk 1]
```diff
@@ -7946,7 +7946,7 @@ def all_env_els_impl(seq=(Bool, True),
         doc="Returns this DefiningName's basic declaration"
     )
 
-    @langkit_property(public=True, return_type=AdaNode.entity.array,
+    @langkit_property(public=True, return_type=T.BaseId.entity.array,
```
Comment (raph-amiard) [inline ada/language/ast.py]: This should be in a separate commit

Comment (raph-amiard) [reply ada/language/ast.py]: lol sorry haha kthxbye

Comment (Roldak) [reply ada/language/ast.py]: Next time I will do one commit per new character
